### PR TITLE
Fix issue #6 

### PR DESCRIPTION
**Pull request by CPatchane**
Replace hardcoded couchdb path by using `get_database_dir()` function.
Issue #6 reported by @clochix

### CODE REVIEW

File: cozy_management/backup.py [hunk 1]
```diff
@@ -22,7 +27,7 @@ def backup():
     cmd += ' --exclude stack.token'
     cmd += ' --exclude couchdb.login'
     cmd += ' --exclude self-hosting.json'
-    cmd += ' /etc/cozy /usr/local/var/cozy /var/lib/couchdb/cozy.couch'
+    cmd += ' /etc/cozy /usr/local/var/cozy '+ COUCHDB_PATH +'/cozy.couch'
```
Comment (nledez) [inline cozy_management/backup.py]: Use `.format` to replace `+`

**Comment by CPatchane** [comment]
String concatenation syntax fixed

**Comment by frankrousseau** [comment]
@nledez Could you review that one?

File: cozy_management/backup.py [hunk 2]
```diff
@@ -22,7 +27,7 @@ def backup():
     cmd += ' --exclude stack.token'
     cmd += ' --exclude couchdb.login'
     cmd += ' --exclude self-hosting.json'
-    cmd += ' /etc/cozy /usr/local/var/cozy /var/lib/couchdb/cozy.couch'
+    cmd += ' /etc/cozy /usr/local/var/cozy {couchdb_path}/cozy.couch'.format(couchdb_path=COUCHDB_PATH)
     cmd = cmd.format(BACKUPS_PATH, timestamp)
```
Comment (nledez) [inline cozy_management/backup.py:31]: @CPatchane can you update these lines too?

```
cmd = 'tar cvzf {}/cozy-{}.tgz'
cmd += ' /etc/cozy /usr/local/var/cozy {couchdb_path}/cozy.couch'.format(couchdb_path=COUCHDB_PATH)
cmd = cmd.format(BACKUPS_PATH, timestamp)
```

```
cmd = 'tar cvzf {backup_path}/cozy-{timestamp}.tgz'
cmd += ' /etc/cozy /usr/local/var/cozy {couchdb_path}/cozy.couch'
cmd = cmd.format(backup_path=BACKUPS_PATH, timestamp=timestamp, couchdb_path=COUCHDB_PATH)
```

Thanks.

**Pull request merged**